javascript/javascript1/week1/Ageify.js

```diff
@@ -0,0 +1,4 @@
+let yearOfBirth = 1987;
+let yearFuture = 2027;
+let age = (yearFuture - yearOfBirth);
+console.log( "You will be " + age + "  years old in " + yearFuture + " .");
```

Comment:
Very clean ans simple code, and nice string concatenation. 👍  The parentheses `()` are not necessary at the subtraction, you could have written:

```javascript
let age = yearFuture - yearOfBirth;
```

But the way you did it is totally fine, actually it is safer that way! 😇